pkg/cassandra/config.go

```diff
@@ -47,11 +47,11 @@ func (c config) MarshalJSON() ([]byte, error) {
 
 	// Even though we default to Cassandra's stock defaults for num_tokens, we need to
 	// explicitly set it because the config builder defaults to num_tokens: 1
-	if c.NumTokens == nil {
+	if c.CassandraYaml.NumTokens == nil {
```

Comment:
This change isn't really necessary as embedded struct fields can be accessed directly. But I agree that this makes the code easier to understand, so 👍 .